start-mongodb.sh

```diff
@@ -47,7 +52,7 @@ echo "::group::Waiting for MongoDB to accept connections"
 sleep 1
 TIMER=0
 
-until docker exec --tty mongodb mongosh --port $MONGODB_PORT --eval "db.serverStatus()" # &> /dev/null
+until docker exec --tty mongodb $MONGO_CLIENT --port $MONGODB_PORT --eval "db.serverStatus()" # &> /dev/null
```

Comment:
```suggestion
until docker exec --tty mongodb mongosh --port $MONGODB_PORT --eval "db.serverStatus()" # &> /dev/null
```
For mongoDB v4 this changes mongosh to mongo. Is that right? It might be why the tests are failing

Reply:
Hey Iain, MongoDB 4.x comes with a `mongo` command. Since MongoDB 5.x, there’s the newer `mongosh` command. The `$MONGO_CLIENT` variable should resolve the correct command depending on the defined MongoDB version.

Reply:
Let me check what happened with the Actions scripts, because locally everything worked, give me some time, probably some mess with shell environment vs my wildcard here:

```
if [[ $MONGODB_VERSION == 4* ]]; then
   MONGO_CLIENT="mongo"
 else
   MONGO_CLIENT="mongosh --quiet"
 fi
```

Working on it...